docs/user_guide/configuration/containerized_server.md

```diff
@@ -487,7 +496,7 @@ Below are the full scripts:
             try:
                 contents = yaml.load(yaml_file, yaml.FullLoader)
             except AttributeError:
-                LOG.warning(
+                print(
```

Comment:
print! we want log don't we?

Reply:
ah I thought I fixed this! I'll go change it again